app/cdap/components/RecipeDetails/DrawerHeader/index.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+import { blue, grey } from '@material-ui/core/colors';
+import React from 'react';
+import styled from 'styled-components';
+import CloseIcon from '@material-ui/icons/Close';
+import DeleteOutlineIcon from '@material-ui/icons/DeleteOutline';
+import EditIcon from '@material-ui/icons/Edit';
+import SaveAltIcon from '@material-ui/icons/SaveAlt';
+import { Box, Typography } from '@material-ui/core';
+import T from 'i18n-react';
+
+interface IDrawerHeaderProps {
+  onCloseDetail: () => void;
+}
+
+const UnderLine = (
```

Comment:
@richadivami 
according to the latest comments added by the client
we should not use SVG icons as react components
we should move these SVGs to the appropriate folder and use

please check if @sivaRajanaDivami is available for support on this
if not, then visit the breadcrumb, connection list, or workspace list PRs to see how @sivaRajanaDivami  has fixed this issue
this has to be updated
also make sure the Underline SVG is not present in the repo already, plz don't create duplicates

Reply:
Done